Description: replace srtp_deinit with srtp_shutdown
Author: Tzafrir Cohen <tzafrir@debian.org>

The internal copy of srtp used in pjmedia has its own srtp_deinit()
instead of srtp_shutdown().

--- a/pjmedia/src/pjmedia/transport_srtp.c
+++ b/pjmedia/src/pjmedia/transport_srtp.c
@@ -323,7 +323,7 @@ static void pjmedia_srtp_deinit_lib(pjme
 
     PJ_UNUSED_ARG(endpt);
 
-    err = srtp_deinit();
+    err = srtp_shutdown();
     if (err != err_status_ok) {
 	PJ_LOG(4, (THIS_FILE, "Failed to deinitialize libsrtp: %s", 
 		   get_libsrtp_errstr(err)));
